src/tests/ftest/pool/create_query.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2021-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from pool_test_base import PoolTestBase
+
+
+class PoolCreateQueryTests(PoolTestBase):
+    # pylint: disable=too-many-ancestors
+    """Pool create tests.
+
+    All of the tests verify pool create performance with 7 servers and 1 client.
+    Each server should be configured with full compliment of NVDIMMs and SSDs.
+
+    :avocado: recursive
+    """
+
+    def test_create_and_query(self):
+        """JIRA ID: DAOS-10339
+
+        Test Description:
+            Create a pool and check that the size of the response is close enough to the requested
+            size.  Verify that the size of pool is equal to the size returned in the response of the
+            pool create.
+
+        :avocado: tags=all,pr,daily_regression
+        :avocado: tags=hw,medium,ib2
+        :avocado: tags=pool
+        :avocado: tags=pool_create_tests,create_and_query
```

Comment:
Optional/Recommendation - Personally, I like to use a unique tag that includes the directory/feature and filename/test case.
```suggestion
        :avocado: tags=pool_create_query
```
That way it's intuitive that `pool_create_query` is in the `pool` directory and runs the `create_query` test. But this is not a requirement.

Reply:
- [ ] Fix tag name

Reply:
Initially it was in the pool test file, and I used its naming convention. However, I agree that your proposition is more intuitive (at least for a DAOS newbie such as me)